model_compression_toolkit/target_platform_capabilities/target_platform/op_quantization_config.py

```diff
@@ -177,6 +258,61 @@ def clone_and_edit(self, **kwargs):
             self.__edit_quantization_configuration(qc, kwargs)
         return qc_options
 
+    def clone_and_edit_weight_attribute(self, attrs: List[str] = None, **kwargs):
+        """
+        Clones the quantization configurations and edits some of their attributes' parameters.
+
+        Args:
+            attrs: attributes names to clone their configurations. If None is provided, updating the configurations
+                of all attributes in the operation attributes config mapping.
+            **kwargs: Keyword arguments to edit in the attributes configuration.
+
+        Returns:
+            QuantizationConfigOptions with edited attributes configurations.
+
+        """
+
+        qc_options = copy.deepcopy(self)
+
+        for qc in qc_options.quantization_config_list:
+            if attrs is None:
+                attrs_to_update = list(qc.attr_weights_configs_mapping.keys())
+            else:
+                if not isinstance(attrs, List):
+                    Logger.error(f"Expecting a list of attribute but got {type(attrs)}.")
+                attrs_to_update = attrs
+
+            for attr in attrs_to_update:
+                if qc.attr_weights_configs_mapping.get(attr) is None:
+                    Logger.error(f'Edit attributes is possible only for existing attributes '
+                                 f'in the configuration weights config mapping, but {attr} is not an attribute of {qc}.')
+                self.__edit_quantization_configuration(qc.attr_weights_configs_mapping[attr], kwargs)
+        return qc_options
+
+    def clone_and_map_weights_attr_keys(self, layer_attrs_mapping: Union[Dict[str, str], None]):
+        """
+       Clones the quantization configuration options and edits the keys in the each configuration attributes config mapping,
+       based on the given attributes names mapping.
+
+        Args:
+            layer_attrs_mapping: A mapping between attributes names.
+
+        Returns:
+            QuantizationConfigOptions with edited attributes names.
+
+        """
+        qc_options = copy.deepcopy(self)
+
+        for qc in qc_options.quantization_config_list:
+            for attr in list(qc.attr_weights_configs_mapping.keys()):
+                if layer_attrs_mapping is None:
+                    qc.attr_weights_configs_mapping = {}
+                else:
+                    qc.attr_weights_configs_mapping[layer_attrs_mapping[attr]] = (
+                        qc.attr_weights_configs_mapping.pop(attr))
```

Comment:
isn't `pop` a bug if 2 `attr `will need to copy from `qc.attr_weights_configs_mapping`?